app/controllers/users_controller.rb

```diff
@@ -0,0 +1,78 @@
+# class to create a new table attendances
+class UsersController < ApplicationController
+  before_action :set_user, only: %i[show edit update destroy]
+  before_action :authorized, only: %i[edit show destroy update]
+
+  # GET /users
+  # GET /users.json
+  def index
+    @users = User.all
+  end
+
+  # GET /users/1
+  # GET /users/1.json
+  def show
+    @events = Event.all
+    set_user
```

Comment:
_set\_user_ here seem to be unnecessary since you have _current\_user_ for use in your views.